db/app.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require("supertest");
+const app = require("../db/app");
+const db = require("../db/index");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+	return seed(data);
+});
+afterAll(() => {
+	return db.end();
+});
+describe("GET /api/topics", () => {
+    test("200 responds with an array of all topics", () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({ body }) => {
+                const { topics } = body;
+                expect(topics).toHaveLength(3)
+                topics.forEach((topic) => {
+                    expect(topic).toHaveProperty("slug", expect.any(String))
+                    expect(topic).toHaveProperty("description", expect.any(String))
+                })
+            })
+    })
```

Comment:
We should have the 404 as a separate describe block as it's not really specific to topics.